buildsystem/HandlePythonOptions.cmake

```diff
@@ -7,17 +7,10 @@ find_package(Python 3.4 REQUIRED)
 find_package(Cython 0.25 REQUIRED)
 find_package(Numpy REQUIRED)
 
-py_get_config_var(OPT PYEXT_CXXFLAGS)
```

Comment:
That's just the compiler flags, I don't think there will be a lot of fancy stuff on other platforms. The python include paths and all the other stuff we query the interpreter for is not affected by this change.